app/controllers/result_filters/subject_controller.rb

```diff
@@ -14,7 +14,12 @@ def new; end
     def start; end
 
     def create
-      redirect_to results_path(filter_params.merge(subjects: convert_rails_subject_id_params_to_csharp))
+      if params[:subjects].blank?
```

Comment:
my bad needs to check if `SEND` 

https://github.com/DFE-Digital/search-and-compare-ui/blob/9130e9538fd2848f87bf4f21da09d44a8227ebd0/src/Controllers/FilterController.cs#L71-L75

Reply:
Good point, I forgot SEND is a separate thing